nightwatch.conf.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
           userName: '${BROWSERSTACK_USERNAME}',
           accessKey: '${BROWSERSTACK_ACCESS_KEY}',
           buildName: 'browserstack-build-1',
+          sessionName : "BStack nightwatch snippet",
```

Comment:
Different session Name for browserstack and browserstack.local

Reply:
changed